db/queries.py

```diff
@@ -1,46 +1,89 @@
+import datetime
+
 from db.models import Message, User, Chat
 from django.db.models import Q, Count, F
 
 
 def get_messages_that_contain_word(word: str) -> list[Message]:
-    pass
+    return list(Message.objects.filter(text__icontains=word))
 
 
 def get_untitled_chats() -> list[Chat]:
-    pass
+    return Chat.objects.filter(
+        title__startswith="Untitled"
+    )
 
 
-def get_users_who_sent_messages_in_2015() -> list[str]:
-    pass
+def get_users_who_sent_messages_in_2015() -> list[tuple]:
+    users = User.objects.all()
```

Comment:
no need of this variable, u can write User.objects.filter... in your list comprehension

Reply:
Also u can make it through the Message model using method "values_list"